src/resources/product/product.controllers.js

```diff
@@ -17,8 +18,18 @@ const createProduct = async (req, res) => {
       price,
       size,
     } = req.body;
+
+    const response = await getCategoryById(categoryId);
+    if (response.length <= 0) {
```

Comment:
is there any case where `response.length` is < 0? If no, we can just do `if (response.length).....` since 0 is falsy in javascript